### PR TITLE
Hotfix_s3_load_and_json_save

### PR DESCRIPTION
**Pull request by threnjen**
Three overall elements:

- Fixes a filename class attributre error in scraper
- Fixes incorrect code to load object from S3
- Dumps json to bytes for write

### CODE REVIEW

**Review by aherrmann85 — COMMENTED** [review]
I'll send up a commit that fixes this, but I want to work on the other repo right now (and will likely need to setup my computer today).

File: game_data_scraper/main.py [hunk 1]
```diff
@@ -162,12 +164,12 @@ def _combine_xml_files_to_master(self) -> str:
 
         return xml_bytes
 
-    def _write_master_xml_file(self, xml_bytes: bytes, filename=str) -> None:
+    def _write_master_xml_file(self, xml_bytes: bytes, xml_filename=str) -> None:
 
-        LocalSaver(XMLWriter(), self.scraped_games_save).save_data(xml_bytes, filename)
+        LocalSaver(XMLWriter(), self.scraped_games_save).save_data(xml_bytes, xml_filename)
```
[the remark this answers is not in the document]
Comment (threnjen) [reply game_data_scraper/main.py:174]: the XML writes are working fine

File: utils/read_write.py [hunk 2]
```diff
@@ -25,8 +26,8 @@ def write_data(cls, data: dict) -> bytes:
 
 
 class JSONWriter(DataWriter):
-    def write_data(cls, data: dict) -> bytes:
-        return json.dumps(data)
+    def write_data(cls, data: Union[dict, list]) -> bytes:
```
[the remark this answers is not in the document]
Comment (threnjen) [reply utils/read_write.py:29]: :)

**Comment by threnjen** [comment]
> I'll send up a commit that fixes this, but I want to work on the other repo right now (and will likely need to setup my computer today).

Everything is currently working

**Pull request merged**